templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
+  url:  <%%= ENV.fetch("DATABASE_URL") %>
+  pool: <%%= ENV.fetch("DB_POOL", ENV.fetch("MAX_THREADS", 5) %>
```

Comment:
Do we want this `pool` setting in all environments? Worth moving to `default` / `development` and then using the default in the deploy environments? Looks like this `pool` setting is currently out of sync with the `development` version. Not sure if that could cause any unexpected bugs in development, but either way, seems like maintaining parity could be valuable.